shell/server/hack-session.js

```diff
@@ -33,6 +33,8 @@ const Grain = Capnp.importSystem("sandstorm/grain.capnp");
 
 const Url = Npm.require("url");
 
+const escapeRegExp = require('lodash.escaperegexp');
```

Comment:
Not completely sure if this is how one should do it in new Meteor. But I will test it once I get to run it. :-)